src/SFML/Graphics/Sprite.cpp

```diff
@@ -118,10 +116,7 @@ const Color& Sprite::getColor() const
 ////////////////////////////////////////////////////////////
 FloatRect Sprite::getLocalBounds() const
 {
-    auto width  = static_cast<float>(std::abs(m_textureRect.width));
-    auto height = static_cast<float>(std::abs(m_textureRect.height));
-
-    return FloatRect({0.f, 0.f}, {width, height});
+    return FloatRect({0, 0}, Vector2f(m_textureRect.size));
```

Comment:
What happened to the `std::abs` logic? Did the behavior of this function change?

Reply:
Fixed